search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -122,7 +123,8 @@ class TemporalEntityHandler(
             jsonLdEntityWithTemporalValues.contexts
         )
 
-        return ResponseEntity.status(HttpStatus.OK).body(serializeObject(compact(filteredJsonLdEntity, contextLink)))
+        return buildGetSuccessResponse(mediaType, contextLink)
+            .body(serializeObject(compact(filteredJsonLdEntity, contextLink)))
```

Comment:
Here we are not supposed to use the mediaType in compaction ?

Reply:
Yes, indeed.

Reply:
Done